### ISSUE TITLE
Socket hang up

### ISSUE THREAD

**Issue by dylanrussellmd** [Pedro-Bronsveld/anki-editor#6]
Hi,

I have been having zero issues with anki-editor until yesterday evening when suddenly anki-editor is giving me the error: "socket hang up". It is unable to retrieve any note types.

anki-editor version 1.0.4
Anki Version ⁨24.04.1 (ccd9ca1a)⁩
Python 3.9.18 Qt 6.6.2 PyQt 6.6.1
Anki Connect is the latest version (re-installed from AnkiWeb yesterday).

Output of `lsof -i :8765 -S`:

```
COMMAND  PID         USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
anki    8793 dylanrussell   84u  IPv4  46810      0t0  TCP localhost:8765 (LISTEN)
```

Thanks for your help.

**Comment by jonnymaceachern** [comment]
I've been using this extension for a few months and only started experiencing this issue as well.

![CleanShot 2024-09-08 at 21 18 19](https://github.com/user-attachments/assets/83e25a75-2769-45c7-8809-af7002ebe9e9)

**Comment by jonnymaceachern** [comment]
@dylanrussellmd 

As a temporary fix, you can revert to [VS Code 1.92](https://code.visualstudio.com/updates/v1_92). I tested it on my end and it's working.

This extension hasn't be updated recently, so it's definitely something to do with VS Code's latest August update.

**Comment by dylanrussellmd** [comment]
> @dylanrussellmd
> 
> As a temporary fix, you can revert to [VS Code 1.92](https://code.visualstudio.com/updates/v1_92). I tested it on my end and it's working.
> 
> This extension hasn't be updated recently, so it's definitely something to do with VS Code's latest August update.

I figured that was the case! Thanks for the temporary fix.

**Comment by Pedro-Bronsveld** [comment]
Hi,

Thanks for raising this issue. It does seem the VSCode update to version 1.93 broke something in how http requests were sent from VSCode to Anki Connect.

The problem should be fixed in version 1.0.5 of anki-editor.
